app/views/support_interface/application_forms/_application_navigation.html.erb

```diff
@@ -1,4 +1,46 @@
+<% content_for :before_content do %>
+  <% if title == 'Details' %>
+    <%= render BreadcrumbComponent.new(items: [
+      {
+        text: 'Candidates',
+        path: support_interface_candidates_path
+      },
+      {
+        text: @application_form.candidate.email_address,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: @application_form.support_reference
+      }
+    ]) %>
+  <% else %>
+    <%= render BreadcrumbComponent.new(items: [
+      {
+        text: 'Candidates',
+        path: support_interface_candidates_path
+      },
+      {
+        text: @application_form.candidate.email_address,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: @application_form.support_reference,
+        path: support_interface_candidate_path(@application_form.candidate)
+      },
+      {
+        text: title
+      }
+    ]) %>
+  <% end %>
+<% end %>
```

Comment:
I think the approach makes sense but you could DRY it:

```suggestion
  <%= render BreadcrumbComponent.new(items: [
    {
      text: 'Candidates',
      path: support_interface_candidates_path
    },
    {
      text: @application_form.candidate.email_address,
      path: support_interface_candidate_path(@application_form.candidate)
    },
    {
      text: @application_form.support_reference,
      path: support_interface_candidate_path(@application_form.candidate)
    }
  ].concat(title == 'Details' ? [{ text: title }] : [])) %>
<% end %>
```

Reply:
A-ha! Awesome, have made these all dryer. Knew they’d be a way. Thanks @tvararu 😃